simple/schema.py

```diff
@@ -387,6 +390,22 @@ class Spectra(Base):
         {},
     )
 
+    @validates("access_url", "regime", "source", "telescope", "instrument", "mode")
+    def validate_required(self, key, value):
+        if value is None:
+            raise ValueError(f"Value required for {key}")
+        return value
+
+    @validates("observation_date")
+    def validate_date(self, key, value):
+        if value is None:
+            raise ValueError(f"Invalid date received: {value}")
+        elif not isinstance(value, datetime):
+            # Convert to datetime for storing in the database
+            # Will throw error if unable to convert
+            value = datetime.fromisoformat(value)
+        return value
```

Comment:
Can we raise a warning saying that converted date from and to?

Reply:
I can do so easily via a print statement, or are you suggesting we import the warnings package and/or logging?

Reply:
Added `print` for now, but we may want to consider using logging since we use it in the ingest scripts.